src/presentation/styles/styledComponents/StyledHeader.js

```diff
@@ -6,14 +6,17 @@ import NLUlogo from '../../assets/NLULogoTransparent.png';
 import { HoverButtonNoClick } from '../framerMotionComponents/HoverButton';
 import { Tab, StyledDiv } from './StyledConstants';
 
-import { getBestIntents } from '../../../controllers/UserRequests';
+import { getBestIntents, getDefaultIntents } from '../../../controllers/UserRequests';
 
 // Define the list of types that serve as the selection of tabs on the left
 //export var types = ['Option A', 'Option B', 'Option C'];
 
-export var types = ['Option A', 'Option B'];
-
+//This is the type of the voiceflow blocks
+export var types = ["Option A", "Option B"];
 
+//TEMPORARY FIX TO CALL GET REQUEST ONCE (no classes): 
+//Variable might need to be a use state variable to work properly 
+var getRequestCalled = false; 
```

Comment:
May need to be moved into a UseState hook later, and then into an "instance attribute" of a React Component class if we get to migrating to classes.